README.md

```diff
@@ -84,3 +84,24 @@ Creating an API token for your Dynatrace environment is described in the [Dynatr
 The scope required for sending metrics is the `Ingest metrics` scope in the **API v2** section:
 
 ![API token creation](docs/img/api_token.png)
+
+#### Metric Key Prefix
+
+The `setPrefix` method of the builder specifies an optional prefix, which is prepended metric key, separated by a dot (`<prefix>.<namespace>.<name>`).
+
+#### Default Dimensions
+
+The `setDefaultDimensions` method can be used to optionally specify a list of key/value pairs (`Label`s), which will be added as additional dimensions to all data points.
+Dimension keys are unique and will be de-duplicated, and only one dimension value per key will be sent to the server.
+
+#### Export OneAgent Metadata
+
+Can be set by adding `setEnrichWithOneAgentMetaData`. If running on a host with a running OneAgent, setting this option will export metadata collected by the OneAgent to the Dynatrace endpoint.
```

Comment:
>Can be set by adding `setEnrichWithOneAgentMetaData`.

This isn't a sentence and I'm not really sure what it means. Is this a configuration property that needs to be set true or false or a function that needs to be called? What is the default?

Reply:
updated it to be more clear